packages/react/src/app-layout/AppLayoutHeaderBrand.tsx

```diff
@@ -9,24 +9,138 @@ import {
 } from '../core';
 import { Text } from '../text';
 import { Box } from '../box';
+import { Stack } from '../stack';
+import { AppLayoutHeaderProps } from './AppLayoutHeader';
+
+const GAP_REM = 1.5;
 
 export type AppLayoutHeaderBrandProps = {
 	href: string;
 	logo: JSX.Element;
+	hasAccountDetails: boolean;
 	heading: string;
-	subLine?: string;
 	badgeLabel?: string;
+	dividerPosition?: AppLayoutHeaderProps['dividerPosition'];
+	secondHref?: AppLayoutHeaderProps['secondHref'];
+	secondLogo?: AppLayoutHeaderProps['secondLogo'];
+	subLine?: string;
 };
 
 export function AppLayoutHeaderBrand({
-	heading,
-	subLine,
 	badgeLabel,
+	dividerPosition,
 	logo,
+	heading,
+	hasAccountDetails,
 	href,
+	secondHref,
+	secondLogo,
+	subLine,
 }: AppLayoutHeaderBrandProps) {
 	const Link = useLinkComponent();
-	return (
+
+	return logo && secondLogo ? (
```

Comment:
The mid breakpoint shouldn't have a different gap if it has a second logo 
![image](https://github.com/agriculturegovau/agds-next/assets/853552/efc884c4-2089-4554-977c-e49f468307ca)